src/ssis_migrator/cli.py

```
@@ -0,0 +1,449 @@
+            console.print(f"❌ Conversion failed!", style="red")
+            console.print(f"Errors: {result.errors}")
+            
+            # Display detailed error report
+            display_error_report(error_handler, verbose=True)
+            
+            sys.exit(1)
+            
+    except Exception as e:
+        error_handler.handle_error(
+            e,
+            context=create_error_context(
```

Flake8 (7.2.0)

[error] undefined name 'create_error_context'

(F821)